### PR TITLE
Fixed invalid link name for ApiControllerAttribute

### LINKED ISSUE #230: Invalid link description in ApiControllerAttribute documentation
**Issue description**
https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.apicontrollerattribute?view=aspnetcore-6.0#remarks has link name `Cookie policy middleware`, however, it actually leads to https://docs.microsoft.com/en-us/aspnet/core/web-api/#apicontroller-attribute.

~~I couldn't find any corresponding file for this doc to make a PR and also it doesn't have the "quick edit" button, so I suppose it was auto-generated.~~

**Target framework**

- [x] .NET Core
- [ ] .NET Framework
- [ ] .NET Standard

### PR DESCRIPTION
**Pull request by KeterSCP**
Closes #230

### CODE REVIEW

**Comment by KeterSCP** [comment]
@Rick-Anderson was this failure caused by my changes or it is unrelated?

**Comment by Rick-Anderson** [comment]
@KeterSCP see #232
While you're at it, get rid of future tense in the PR.

File: aspnet-core/xml/Microsoft.AspNetCore.Mvc/ApiControllerAttribute.xml [hunk 1]
```diff
@@ -56,9 +56,8 @@
             developer experience for building APIs.
             </para><para>
             When decorated on an assembly, all controllers in the assembly will be treated as controllers with API behavior.
```
Comment (Rick-Anderson) [inline aspnet-core/xml/Microsoft.AspNetCore.Mvc/ApiControllerAttribute.xml:58]:
```suggestion
            When decorated on an assembly, all controllers in the assembly are treated as controllers with API behavior.
```

File: aspnet-core/xml/Microsoft.AspNetCore.Mvc/ApiControllerAttribute.xml [hunk 2]
```diff
@@ -55,10 +55,9 @@
             Controllers decorated with this attribute are configured with features and behavior targeted at improving the
             developer experience for building APIs.
             </para><para>
-            When decorated on an assembly, all controllers in the assembly will be treated as controllers with API behavior.
-            For more information, see <see href="https://docs.microsoft.com/aspnet/core/web-api/#apicontroller-attribute">ApiController attribute</see>.
```
Comment (Rick-Anderson) [inline aspnet-core/xml/Microsoft.AspNetCore.Mvc/ApiControllerAttribute.xml:59]: Revert this

Comment (KeterSCP) [reply aspnet-core/xml/Microsoft.AspNetCore.Mvc/ApiControllerAttribute.xml:59]: So this is a dupe I was talking about. It is also described in the `Remarks` sections

**Pull request closed without merging**